pset-iterators.js

```diff
@@ -87,6 +96,12 @@ Tip: your new array should contain 45 elements
 console.log('Problem 2:')
 
 // Add your code below this line
+const presidentialParties = []
```

Comment:
@rvaswani85 same as before, `forEach()` works here but `.map()` is the preferred method in this case